src/plugins/visualizations/public/embeddable/visualize_embeddable.ts

```diff
@@ -331,13 +331,14 @@ export class VisualizeEmbeddable extends Embeddable<VisualizeInput, VisualizeOut
       this.abortController.abort();
     }
     this.abortController = new AbortController();
+    const abortController = this.abortController;
```

Comment:
why creating local variable ? if we have it lets use it everywhere

Reply:
local variable here is needed as a reference to previous `abortController` instance. 
when await buildPipeline is finished, in case it was aborted, `this.abortController` would be a reference from _next_ request